governance/governance.md

```diff
@@ -0,0 +1,127 @@
+# Helm Governance
+
+The following document outlines how the Helm project governance operates.
+
+## The Helm Project
+
+The Helm project is made up of several codebases and services with different release cycles that enable users to create, distribute, and operate packages in Kubernetes. These codebases include:
+
+* Helm - the package manager
+* Charts - community curated charts
+* Monocular - a web interface for browsing one or more chart repositories
+* Chartmuseum - a chart repository with support for pushing charts, auth, and more
+* chart-testing - container based chart testing tools
+
+The services provided include:
+
+* Documentation for those who want to create, distribute, depend on, and operate packages
+* Public chart search and discovery
+
+## Maintainers Structure
+
+There are two levels of maintainers for Helm. The Helm org maintainers oversee the overall project and it's health. Project maintainers focus on a single codebase or a group of related codebases. For example, the chart maintainers manage the charts repository and a few repositories that support charts. The Helm org maintainers are responsible for the overall health of the project while the project maintainers are responsible for maintaining the code.
+
+Changes in maintainership have to be announced on the [helm mailing list](https://lists.cncf.io/g/cncf-helm).
+
+### Helm Org Maintainers
+
+The Helm Org maintainers are responsible for:
+
+* Maintaining the mission, vision, values, and scope of the project
+* Refining the governance and charter as needed
+* Making project level decisions
+* Resolving escalated project decisions when the subteam responsible is blocked
+* Managing the Helm brand
+* Controlling access to Helm assets such as source repositories, hosting, project calendars
+* Handling code of conduct violations
+* Deciding what sub-groups are part of the Helm project
+* Overseeing the resolution and disclosure of security issues
+* Managing financial decisions related to the project
+
+Changes to org maintainers use the following:
+
+* There will be between 3 and 9 people (maintaining an odd number). If there is a loss of a org maintainer bring the count to an even number the org maintainers can continue with an even number as long as they are actively going through the process to bring the maintainer count back to an odd number
+  * To ensure diverse representation from all areas of the Helm org, the breakdown of Helm Org Maintainers group will be as follows:
+    * When there are 3 Org Maintainers
+      * 1 Representative from the Helm core project
+        * Also serves as the default chair unless someone else in the Org Maintainers group is interested in the role
+      * 1 Representative from the Charts project
+      * 1 Representative from another Helm project
+    * When there are 5 Org Maintainers
+      * 1 Chair (from any project under the Helm org)
```

Comment:
The election process does not signify how we elect a chair and is not specific on how we figure out project designations. My proposal:

* Highest vote getter who wants to accept the chair position and responsibilities gets it
* The same way we go down popularity and only so many companies can be represented we look at projects.

Thoughts?

Reply:
I think that works.